From d05a0208e30485f0b1d5dd1b4e1d7b06fb9350f8 Mon Sep 17 00:00:00 2001
From: Riajul Islam <riajul@kahf.co>
Date: Tue, 14 Apr 2026 06:27:05 +0000
Subject: [PATCH 16/17] downstream:
 0016-fix-relay-install-perl-and-make-for-vendored-openssl


diff --git a/crates/relay-tunnel/Dockerfile b/crates/relay-tunnel/Dockerfile
index e56f409f5..09287adc6 100644
--- a/crates/relay-tunnel/Dockerfile
+++ b/crates/relay-tunnel/Dockerfile
@@ -6,7 +6,7 @@ ENV CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
 ENV CARGO_TARGET_DIR=/app/target
 
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends pkg-config libssl-dev ca-certificates \
+  && apt-get install -y --no-install-recommends pkg-config libssl-dev ca-certificates perl make \
   && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /app
-- 
2.50.1

