From 8ab688c687d91cf2d031291e2016a8b05d29efbe Mon Sep 17 00:00:00 2001
From: Riajul Islam <kmriajulislami@gmail.com>
Date: Thu, 12 Mar 2026 15:46:55 +0000
Subject: [PATCH] update: loops template IDs

---
 crates/remote/src/mail.rs | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/crates/remote/src/mail.rs b/crates/remote/src/mail.rs
index 83aac904c..a4a8b3b67 100644
--- a/crates/remote/src/mail.rs
+++ b/crates/remote/src/mail.rs
@@ -4,9 +4,9 @@ use api_types::MemberRole;
 use async_trait::async_trait;
 use serde_json::json;
 
-const LOOPS_INVITE_TEMPLATE_ID: &str = "cmhvy2wgs3s13z70i1pxakij9";
-const LOOPS_REVIEW_READY_TEMPLATE_ID: &str = "cmj47k5ge16990iylued9by17";
-const LOOPS_REVIEW_FAILED_TEMPLATE_ID: &str = "cmj49ougk1c8s0iznavijdqpo";
+const LOOPS_INVITE_TEMPLATE_ID: &str = "cmlb3zqwe03960i159htyb93d";
+const LOOPS_REVIEW_READY_TEMPLATE_ID: &str = "cmlb45ki603ia0i1pdiq2ekyr";
+const LOOPS_REVIEW_FAILED_TEMPLATE_ID: &str = "cmlb44vqe03p90i04bwkfqjv3";
 
 #[async_trait]
 pub trait Mailer: Send + Sync {
-- 
2.50.1

